homeassistant/components/xiaomi_miio/device.py

```diff
@@ -89,3 +92,35 @@ def device_info(self):
             device_info["connections"] = {(dr.CONNECTION_NETWORK_MAC, self._mac)}
 
         return device_info
+
+
+class XiaomiCoordinatedMiioEntity(CoordinatorEntity, XiaomiMiioEntity):
```

Comment:
Let's not inherit from `XiaomiMiioEntity` for now and copy its properties to `XiaomiCoordinatedMiioEntity`. We can clean up and refactor later.

Reply:
Done